template/src/features/counter/Counter.test.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @jest-environment jest-environment-jsdom-sixteen
```

Comment:
This will be required till there's a release of CRA with jest 25 - it has been merged to master, but not released.

See MutationObserver shim removed. here - https://github.com/testing-library/dom-testing-library/releases/tag/v7.0.0